builtin/providers/aws/resource_aws_iam_server_certificate.go

```diff
@@ -116,15 +120,15 @@ func resourceAwsIAMServerCertificateCreate(d *schema.ResourceData, meta interfac
 	}
 
 	log.Printf("[DEBUG] Creating IAM Server Certificate with opts: %s", createOpts)
-	resp, err := conn.UploadServerCertificate(createOpts)
+	_, err := conn.UploadServerCertificate(createOpts)
 	if err != nil {
 		if awsErr, ok := err.(awserr.Error); ok {
 			return fmt.Errorf("[WARN] Error uploading server certificate, error: %s: %s", awsErr.Code(), awsErr.Message())
 		}
 		return fmt.Errorf("[WARN] Error uploading server certificate, error: %s", err)
 	}
 
-	d.SetId(*resp.ServerCertificateMetadata.ServerCertificateId)
+	d.SetId(sslCertName)
```

Comment:
Is there a way to get that resource ID? It looks to me like it is set by AWS and I am not sure how to retrieve it.